test/testProject.js

```diff
@@ -478,6 +478,34 @@ module.exports.project = {
         test.ok(project.getOutputLocale("pt-BR"), "pt-BR");
         test.ok(project.getOutputLocale("de-DE"), "de-DE");
 
+        test.done();
+    },
+    testGetOutputLocaleInherit: function(test) {
+        test.expect(2);
+
+        var project = ProjectFactory('./testfiles', {
+            localeInherit: {
+                "en-AU": "en-GB",
+                "en-CN": "en-GB"
+            }
+        });
+        test.ok(project.getLocaleInherit("en-AU"), "en-GB");
+        test.ok(project.getLocaleInherit("en-CN"), "en-GB");
+
+        test.done();
+    },
+    testGetOutputLocaleInheritEmpty: function(test) {
+        test.expect(2);
+
+        var project = ProjectFactory('./testfiles', {
+            localeInherit: {
+                "en-AU": "en-GB",
+                "en-CN": "en-GB"
+            }
+        });
+        test.equals(project.getLocaleInherit("ko-KR"), undefined);
+        test.equals(project.getLocaleInherit(), undefined);
+
         test.done();
```

Comment:
How about a test where the `localeInherit` is not specified in the project constructor? Does getLocaleInherit not fail? What happens if you do `project.getLocaleInherit(locale)` where `locale` is a `Locale` instance or `null`?